.github/actions/prerequisite-nightly/action.yml

```diff
@@ -18,6 +18,8 @@ runs:
         token: ${{ inputs.token }}
 
     - name: Install Rust Nightly
-      uses: ./.github/actions/install-rust-nightly
```

Comment:
This github actions is using a deprecated `set-output`:
```
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files.
```

Replaced with dtolnay